### DISCUSSION TITLE
Addition for projections tab to help understand projection quality

### DISCUSSION THREAD

**Discussion by drorbar** [tanaylab/MCView#154] [General]
Two things me and Oren thought about: 
1. In the projected fold tab - we should have a tick which allows us to see only fitted genes, without it this is a bit hard to use this tab for further decision-making. 
2. In the query tab, we have a table called "Gene metadata", which shows the correction factor. It should also show the correlation between this gene in the atlas and in the query, which is what we care about 

@orenbenkiki - did we have anything else that we thought about?

**Comment by aviezerl** [comment]
1. Fitted in which cell type? @orenbenkiki - Is there somewhere a definition of 'global' fitted genes? Where?
2. @orenbenkiki can you point me to the location of this in the anndata?

**Comment by orenbenkiki** [comment]
It is per cell type but you can OR them

**Comment by orenbenkiki** [comment]
There should also be a global fitted mask

**Reply by aviezerl** [reply]
and what about 2? Is there some correlation measure of query and atlas per gene?

**Comment by orenbenkiki** [comment]
Ugh, good point. I’ll have to compute it and add to the results.

**Comment by orenbenkiki** [comment]
I pushed an update that computes the projected_correlation per gene (in
addition to per metacell).
As for the fitted genes mask, you were right the 1st time - it is only per
type - you have to OR them.
